### PR TITLE
Added Github Templates

### LINKED ISSUE #3: Need of GitHub templates
We need to have Issue Template and PR Template for the project so that the content is uniform and there is a framework for contributors to file an Issue/PR.

- [ ] Issue Template
- [ ] PR Template

### PR DESCRIPTION
**Pull request by ParthS007**
Fixes #3

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+#### Checklist
+
+- [ ] I have read the Contribution & Best practices Guide and my PR follows them.
+- [ ] My branch is up-to-date with the Upstream `master` branch.
+- [ ] I have added necessary documentation (if appropriate)
+
+#### Changes proposed in this pull request:
+
+-
+-
```
Comment (ritikrishu) [inline .github/PULL_REQUEST_TEMPLATE.md]: would be nice to have a _yes/no_ question, if this PR introduces a breaking change (e.g. some breaking API change on the server or client SDK).

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->
+Fixes #
+
+#### Checklist
+
+- [ ] I have read the Contribution & Best practices Guide and my PR follows them.
+- [ ] My branch is up-to-date with the Upstream `master` branch.
+- [ ] I have added necessary documentation (if appropriate)
+
```
Comment (ritikrishu) [inline .github/PULL_REQUEST_TEMPLATE.md]: [ ] I have added/updated necessary tests (if applicable)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ParthS007) [reply .github/PULL_REQUEST_TEMPLATE.md]: I think we can leave this to the contributor.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (ParthS007) [reply .github/PULL_REQUEST_TEMPLATE.md]: correcting :+1:

**Comment by ParthS007** [comment]
Merging it :+1:

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->
+Fixes #
+
+#### Checklist
+
+- [ ] I have read the Contribution & Best practices Guide and my PR follows them.
+- [ ] My branch is up-to-date with the Upstream `master` branch.
+- [ ] I have added/updated necessary tests/documentation (if applicable)
```
Comment (palnabarun) [inline .github/PULL_REQUEST_TEMPLATE.md]: I think decoupling tests and documentation would be a good idea as those are two very important units for sustainable development.

**Comment by realslimshanky** [comment]
Nice work @ParthS007, you rock! :tada:

**Pull request merged**